### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Micky373**
Hello Reviewer,

In this project I have done these tasks:

- Used Webpack (HTML package, CSS rules, and javascript module)
- Used JavaScript to populate the task section
- Used the index.html as a template to build my index.html file in the dist folder
- Deploy the project
- No linter errors
- Fixed all CR's issues

Thanks 😁

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import './styles.css';
+// import checkBoxing from './check_box.js'
+// Populating the html element
```
Comment (Deepakdanger) [inline src/index.js]: - [x] I would recommend removing the commented code to make the codebase cleaner.

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  width: fit-content;
+  border: none;
+  outline: none;
+  margin-left: 1rem;
+}
+
+.task_container {
+  display: flex;
+
+  /* padding: 1rem 0; */
```
Comment (Deepakdanger) [inline src/styles.css]: - [x] Same here. I would recommend removing the commented code to make the codebase cleaner.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+
+## Built With
+
+- JavaScript (ES6)
+- HTML and CSS
+- Webpack
+
+**Instructions on setting up your project locally.**
+The project could be accessed by visiting the website link.
+
```
Comment (Deepakdanger) [inline README.md]: - [x] I would recommend adding all the steps to run the project locally. It should include How to clone, going to right folder, and running right command or opening right file to see the project.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Micky373, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — COMMENTED** [review]
Hi @Micky373, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**